app/src/main/res/drawable/ic_check.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I think, we already have the same file as `ic_dome.xml`. Feel free to update / rename the old file to match the material icon names